### PR TITLE
Include text for moves that drain HP

### PR DESCRIPTION
**Pull request by Ntonio36**
Shows up just near the damage percentages for the concerned moves.

### CODE REVIEW

File: js/index_controls.js [hunk 1]
```diff
@@ -40,23 +40,36 @@ function calculate() {
         result.damageText = minDamage + "-" + maxDamage + " (" + minDisplay + " - " + maxDisplay + notation + ")";
         result.koChanceText = p1.moves[i].bp === 0 ? 'nice move'
                 : getKOChanceText(result.damage, p1, p2, field.getSide(1), p1.moves[i].hits, p1.ability === 'Bad Dreams');
+        var additionalText = '';
```
Comment (AustinXII) [inline js/index_controls.js]: recoveredText?

Comment (AustinXII) [reply js/index_controls.js]: Or even recoveryText

File: js/index_controls.js [hunk 2]
```diff
@@ -40,23 +40,36 @@ function calculate() {
...
         $(resultLocations[0][i].move + " + label").text(p1.moves[i].name.replace("Hidden Power", "HP"));
-        $(resultLocations[0][i].damage).text(minDisplay + " - " + maxDisplay + notation);
+        $(resultLocations[0][i].damage).text(minDisplay + " - " + maxDisplay + notation + additionalText);
         if (maxDamage > highestDamage) {
             highestDamage = maxDamage;
             bestResult = $(resultLocations[0][i].move);
         }
         
         result = damageResults[1][i];
+		additionalText = '';
```
Comment (AustinXII) [inline js/index_controls.js]: Again what Is your formatting

File: js/index_controls.js (hunk 1 above)
Comment (Ntonio36) [reply js/index_controls.js]: Nope, I'm still getting familiarized with GitHub, I didn't dive into merging / squashing commits (amongst many other things).

Comment (AustinXII) [reply js/index_controls.js]: Fwiw  I’m not going to merge two commits, learning how to squash and at least edit multiple files on git is a good thing to learn.

Comment (Ntonio36) [reply js/index_controls.js]: Can you close that pull request so I clean start everything back...by providing multiple files in a single PR ? :D

Comment (Ntonio36) [reply js/index_controls.js]: Never mind ^^

**Pull request closed without merging**